pkg/cmd/cli/nodeagent/server.go

```diff
@@ -419,7 +419,7 @@ func (s *nodeAgentServer) markDataDownloadsCancel(r *controller.DataDownloadReco
 
 func (s *nodeAgentServer) markInProgressPVBsFailed(client ctrlclient.Client) {
 	pvbs := &velerov1api.PodVolumeBackupList{}
-	if err := client.List(s.ctx, pvbs, &ctrlclient.MatchingFields{"metadata.namespace": s.namespace}); err != nil {
+	if err := client.List(s.ctx, pvbs, &ctrlclient.ListOptions{Namespace: s.namespace}); err != nil {
```

Comment:
Why did you think , use ```ctrlclient.ListOptions{``` instead of ```ctrlclient.MatchingFields{``` can do anything better? 

Like the docs note [client#hdr-Indexing](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/client#hdr-Indexing) ,  ```ctrlclient.MatchingFields``` will better than ```ctrlclient.ListOptions{```  when it is used in index condition.

Reply:
This is used to address the error reported in the Velero with a limited permission scenario.

```
time="2023-10-09T23:56:44Z" level=error msg="failed to list backups" error="backups.velero.io is forbidden: User \"system:serviceaccount:arthur:velero-server\
" cannot list resource \"backups\" in API group \"velero.io\" at the cluster scope" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/cmd/server/server.g
o:954" error.function=github.com/vmware-tanzu/velero/pkg/cmd/server.markInProgressBackupsFailed logSource="pkg/cmd/server/server.go:954"
```

I suppose this error is reported, instead of reading from cached, is due to the namespaced cache was not created before use.

Reply:
``` is forbidden: User \"system:serviceaccount:arthur:velero-server ``` , I'm afraid  cache have or have not , may not the error's root cause.😀😀

Reply:
It's related to the limited permission scenario.
Because the query parameter already has the namespace, I didn't expect the query to be in the cluster scope.
I'm not sure that `MatchingFields` will definitely query resources in the whole cluster or query according to the parameter setting.
`ListOptions` doesn't have the error reported issue.